.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Docs 
+*.md @citmc @cmwaters @nooomski @coldice 
+
+# Primary repo maintainers
+* @coldice @gioelebonifetto @8bitpal 
```

Comment:
```suggestion
* @coldice @xavierlepretre @gioelebonifetto @8bitpal 
```
Also here, adding Xavier

Reply:
@okwme wdyt, do we need someone from IG/AiB on this list as well?

Reply:
@toschdev and I struggled for so long as the only two approvers... 
hi @okwme !! 
CODEOWNERS assigns the reviewer automatically. You can choose to add reviewers at will to any PR.

Reply:
Yeah ok cool let's just go ahead with this and iterate when needed.